src/components/NavBar.tsx

```diff
@@ -102,6 +103,16 @@ export default function NavBar({ organization, setShowNavBar }: NavBarProps): JS
         }}
         id='monitoring'
       />
+      <NavSection title={strings.SEEDLINGS.toUpperCase()} />
+      <NavItem
+        label={strings.INVENTORY}
+        icon='iconSeedling'
+        selected={!!isNurseryRoute}
+        onClick={() => {
+          closeAndNavigateTo(APP_PATHS.NURSERY);
+        }}
+        id='nursery'
+      />
```

Comment:
We don't want this displayed for everyone now I think... might be good to use the routeEnabled check.
https://github.com/terraware/terraware-web/blob/main/src/features.ts#L82
```
isRouteEnabled('Nursery management') && (...)
```

Reply:
oops should be featureEnabled not route enabled (basically if user opted in to see this feature).
routeEnabled check is for the routing in App.tsx